src/bp/ui-shared/src/Form/SingleControl.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react'
+
+import Form from '.'
+import { SingleControlProps } from './typings'
+
+const SingleControl: FC<SingleControlProps> = props => {
+  const TEMP_FIELD = 'field'
```

Comment:
I don't get it why always use that nested key ?

Reply:
I added this as a simple wrapper when you want to use a single field with the form controls. For example, in the say something form, there is a dropdown to select the content type, and depending on what you choose here, it displays the form for the selected type.

```js
// Instead of this
<Form.Form
  fields={{
    title: {
      type: ControlType.String,
      title: 'title'
    },
  }}
  formData={{ title: formData?.title }}
  onUpdate={onUpdate}
/>

// you can use it simply like this
<Form.SingleControl
  control={{
    type: ControlType.String,
    title: 'title'
  }}
  value={title}
  onChange={newTitle => console.log(newTitle)}
/>
```